explorer/client/src/components/ownedobjects/views/OwnedCoinView.tsx

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Coin } from '@mysten/sui.js';
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { ReactComponent as ContentIcon } from '../../../assets/SVGIcons/closed-content.svg';
+import { handleCoinType } from '../../../utils/stringUtils';
+import Longtext from '../../longtext/Longtext';
+import Pagination from '../../pagination/Pagination';
+import { type DataType, ITEMS_PER_PAGE } from '../OwnedObjectConstants';
+
+import styles from '../styles/OwnedCoin.module.css';
+
+export default function OwnedCoinView({ results }: { results: DataType }) {
+    const CLOSED_TYPE_STRING = '';
```

Comment:
Rather than empty string being closed I'd personally prefer this flip types between `null` and `string` so that it's a little more obvious, but not a blocking issue.

Reply:
Implemented